about.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./about.css">
+    <!-- Fontawesome Icons -->
+    <script src="https://kit.fontawesome.com/4e655bdddb.js" crossorigin="anonymous"></script>
+    <title>About | Programmer Summit</title>
+</head>
+
+<body>
+    <!-- Header Section Starts Here -->
+    <header>
+        <img id="menu-icon" class="menu-icon" src="./assets/menu-mobile.png" alt="">
+        <ul class="header-head">
+            <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+        
+        <nav id="navbar" class="header-body">
+            <i id="mobile-menu-close-icon" class="mobile-menu-close-icon fa-solid fa-x"></i>
+            <a class="logo" href="index.html"><img src="./assets/logo-dark.png" alt="logo"></a>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="">Programe</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href=""><button class="header-btn">CC Campaign</button></a></li>
+            </ul>
+        </nav>
+
+    </header>
+    <!-- Header Section Ends Here -->
+
+    <!-- Headline Section Starst Here -->
+    <section class="headline">
+        <h6 class="headline__greetings-heading">"Hello! Sharing world"</h6>
+        <h1 class="headline__main-heading">Programmer Summit 2023</h1>
+        <p class="headline-description">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quo temporibus minus
+            sequi officiis incidunt odio libero a neque magnam voluptatum expedita eaque, sunt voluptatibus ab!
+            Accusamus expedita velit vitae quasi.
+            Nulla quisquam autem, in, magnam fugiat porro iure.</p>
+        <h6 class="headline-address">Please contact us per Email for any firther questions about Programmer Summit 2023
+        </h6>
+        <h6 class="headline-email">email@domain.com</h6>
+    </section>
+    <!-- Headline Section Ends Here -->
+
+    <!-- Logo Selection Section Starts Here -->
+    
+    <section class="logo-selection">
+        <h3>Programmer Summit 2023 Logo</h3>
+        <hr>
+        <p>The logo of Programmer Summit 2023 was decided through logo competition from 7. July - 12. July</p>
+        <div class="logo-selection__logo">
+            <img src="./assets/logo-dark.png" alt="">
+        </div>
+    </section>
+    <hr class="hr-full">
```

Comment:
- [ ] Kindly consider matching your design with Behance by choosing the correct font, padding, and margin

| Your design  | Behance design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/217607996-78fad2d2-6494-4fa6-9b2e-e9ad75197623.png)  | ![image](https://user-images.githubusercontent.com/53402600/217608128-74937e30-4dfa-4ff0-9825-2ebc3a6d1b92.png) |